src/index.js

```diff
@@ -527,37 +534,21 @@ Photo.propTypes = {
 }
 function Photo({
   photo,
-  width,
```

Comment:
I'm happy to see the change to CSS grid here! I'm no expert, but it seems to solve a lot of common problems and the designers seem happy with it, so I'm on board 🙂 

At a high level, my feedback to this PR centers around the question: how much can we trust CSS grid to correctly size images for us? And I think this particular line is what sent me down that rabbit hole.

There's a lot of code in this file dedicated to sizing images properly within the grid. Now that we're setting width based on the grid, can we do the same thing with the height? I'd love to see us lean more heavily on CSS and get away from some of our custom sizing JS

Reply:
I think we can trust CSS Grid to do a good job sizing the width/columns accurately. However, in playing with this over the last few days, I ran into a lot of issues with sizing the rows/height. The issue seems to be that without an explicit height set on the image, it breaks out of its container and any CSS Grid Rows. 

However, we can use the same solution of passing the calculated `searchResultHeight` to a CSS variable and use it in a named class on the image. This can cut down on some JS in the form of props being used and clean up some styles that are directly applied to the image. 

The incoming commits include these proposed changes, along with some additional details.